website/static/js/projectSettings.js

```diff
@@ -107,7 +107,40 @@ var ProjectSettings = oop.extend(
                     }
                 }
             };
-        }
+        },
+        enableMailingList: function() {
+            var request = $.ajax({
+                type: 'post',
+                dataType: 'json',
+                url: nodeApiUrl + 'mailing_list/'
+            });
+            request.done(function() {
+                window.location.reload();
+            });
+            request.fail($osf.handleJSONError);
+        },
+        disableMailingList: function(nodeType) {
+
+            var message = '<p>Are you sure that you want to disable the mailing list for this ' + nodeType + '?';
```

Comment:
I don't find any other instances of `disableMailingList` in the codebase, nor `enableMailingList`. Why are these functions here? And—since it appears to work—what is the UI using instead of these functions?